doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,351 +3,293 @@
 
 This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
+ifdef::PAM[]
 
-== {CENTRAL}
-
-.Guided rules expose parentheses outside of an expression [https://issues.jboss.org/browse/RHDM-1046[RHDM-1046]]
+== Installation
 
-Issue: When you create a guided rule, if you select the field for multiple values and each value is enclosed in
-parenthesis, for example `(red), (blue)`,  an invalid rule is generated.
+.{PRODUCT} in {JWS} data source configuration is missing elements [https://issues.jboss.org/browse/RHPAM-2428[RHPAM-2428]]
 
+Issue: When you install {PRODUCT} in {JWS} and configure a data source, there are some missing elements that are needed for the setup to work properly.
 
 Steps to reproduce:
 
-. Open {CENTRAL}.
-. Open any project.
-. Create the data object `Car` with the field `color:String`.
-. Create a new guided decision table.
-. Define a new *Condition* column:
-+
-*Pattern:* `$c : Car`
-+
-*Calculation Type*: `Literal value`
-+
-*Field*: `color`
-+
-*Operator*: `is contained in the (comma separated) list`
-+
-*Header*: `Car color is`
-. Select the *Model* tab and enter `(red), (blue)` in the corresponding cell.
-
-Expected result: The `$c : Car(color in ("(red)", "(blue)"))` rule is generated.
+. Install {PRODUCT} on {JWS}.
+. Configure a database.
```

Comment:
I like the one that is in different issues
`When prompted, click *Configure Advanced Properties* -> *Configure Database Settings*`